common/middleware/get-payment-request.tests.js

```diff
@@ -0,0 +1,104 @@
+const path = require('path')
+const sinon = require('sinon')
+const assert = require('assert')
+const proxyquire = require('proxyquire')
+const _ = require('lodash')
+const paymentFixtures = require('../../test/fixtures/payments-fixtures')
+
+describe('Get payment request middleware', function () {
+  it('should retrieve a payment request if present in the session', function () {
+    let paymentRequestExternalId = 'asdasdkjshfjdks'
+    let returnUrl = 'http://bla.test'
+    let gatewayAccountId = 23
```

Comment:
lack of `const` is causing `const`-ernation :-)

Reply:
I should stop writing swift :D